src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -504,17 +504,17 @@
   <data name="PX1020Fix" xml:space="preserve">
     <value>Make the string length sufficient for the auto-numbering</value>
   </data>
-  <data name="PX1022DacFix" xml:space="preserve">
+  <data name="PX1022DacExtensionFix" xml:space="preserve">
```

Comment:
@EAndrosova please review changes in the PX1022 diagnostic messages and code fix messages